metadata/index.html

```diff
@@ -1453,11 +1467,12 @@
           What should be the mapping of an empty cell? 
         </p>
         <ol>
-          <li>unless the <code>datatype</code> is <code>string</code> or <code>anySimpleType</code> or <code>any</code>, strip leading and trailing whitespace from the value</li>
+          <li>unless the <a href="#cell-valueUrl"><code>valueUrl</code></a> is defined on the cell, <code>datatype</code> is <code>string</code> or <code>anySimpleType</code> or <code>any</code>, strip leading and trailing whitespace from the value</li>
           <li>if the value is the same as the <code>null</code> value, then the value is <code>null</code></li>
           <li>if the <code>separator</code> property is not <code>null</code>, create a list of values by splitting the string at the character specified by the <code>separator</code> property</li>
           <li>validate the value(s) against the <code>format</code>, if one is specified, as described below; raise an error if any of the values do not match the specified format</li>
-          <li>parse the value(s) using the <code>format</code>, as described below, to give the semantic value(s) for the cell</li>
+          <li>if the cell column has a <a href="#cell-valueUrl"><code>valueUrl</code></a> the value(s) are the result of applying the URI template to each value as a <cite><a href="http://www.w3.org/TR/rdf11-concepts/#dfn-resource">resource</a></cite></li>
```

Comment:
Don't reference RDF resources here (this isn't something RDF specific), just say that the values are URLs.